internal/wasm/interpreter/interpreter.go

```diff
@@ -1670,6 +1669,17 @@ func (ce *callEngine) callNativeFunc(ctx context.Context, callCtx *wasm.CallCont
 	ce.popFrame()
 }
 
+// popMemoryOffset takes a memory offset off the stack for use in load and store instructions.
+// As the top of stack value is 64-bit, this ensures it is in range before returning it.
+func (ce *callEngine) popMemoryOffset(op *interpreterOp) uint32 {
+	// TODO: Document what 'us' is and why we expect to look at value 1.
```

Comment:
@mathetake I want to backfill a test for this, but before I do, can you help me answer this?

Reply:
that's because we holds the offset constant there for memory operations 
https://github.com/tetratelabs/wazero/blob/main/internal/wasm/interpreter/interpreter.go#L357-L393